tests/components/mobile_app/const.py

```diff
@@ -28,6 +28,21 @@
     'device_name': 'Test 1',
     'manufacturer': 'mobile_app',
     'model': 'Test',
+    'os_name': 'Linux',
+    'os_version': '1.0',
+    'supports_encryption': True
+}
+
+REGISTER_BAD_COMPONENT = {
```

Comment:
Why do you store this in a constant in a different file ? What are the odds this will be reused?

Reply:
And we can always consider generalizing it _when_ we re-use it.